utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -1159,6 +1181,25 @@ public static boolean validateGuestCidr(final String cidr) {
         }
     }
 
+    public static boolean validateGuestCidr(final CIDR cidr) throws BadCIDRException {
```

Comment:
Why is this method implemented here rather on ``CIDR`` for IPv4/IPv6 dual stack operation?